tests/fixtures/__init__.py

```diff
@@ -72,18 +89,45 @@ def non_mocked_hosts() -> list[str]:
     return ["test"]
 
 
-@async_fixture(name="client")
-async def fixture_client(mongodb_fixture) -> AsyncGenerator[AsyncClient, None]:
-    """Get test client for the work package service"""
+async def publish_datasets(container: Container) -> None:
+    """Publish datasets to populate the database."""
+    config = container.config()
+    async with KafkaEventPublisher.construct(config=config) as event_publisher:
+        await event_publisher.publish(
+            payload=DATASET_OVERVIEW_EVENT.dict(),
+            type_="metadata_dataset_overview",
+            key="test_key",
+            topic="metadata",
+        )
+
+    event_subscriber = await container.event_subscriber()
+    await asyncio.wait_for(event_subscriber.run(forever=False), timeout=5)
```

Comment:
In the future, please consider using the KafkaFixture.publish_event provided by hexkit: https://github.com/ghga-de/hexkit/blob/main/hexkit/providers/akafka/testutils.py#L352

Added to the refactoring list.

Reply:
Already done in ed11dcb since this was a very simple change.